app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.READ_CALENDAR" />
```

Comment:
Do we also need the permission to write to calendar?

Reply:
I don't think that we will be writing to the calendar in any of our features...

Reply:
Ok, I think it was for a future enter to calendar for quick link access to next class directions but I think that we can table that feature for now